stable/sonarqube/templates/ingress.yaml

```diff
@@ -1,7 +1,7 @@
 {{- if .Values.ingress.enabled -}}
 {{- $serviceName := include "sonarqube.fullname" . -}}
 {{- $servicePort := .Values.service.externalPort -}}
-apiVersion: extensions/v1beta1
```

Comment:
Maybe use capabilities https://helm.sh/docs/chart_template_guide/builtin_objects/ in order to use a suitable version? Then it can fit on older and newer clusters.

Reply:
Done. If the new API Version is available it's used. If not, the old API Version is used.